src/pybind/mgr/dashboard_v2/module.py

```diff
@@ -156,6 +156,7 @@ def handle_command(self, cmd):
     def notify(self, notify_type, notify_id):
         NotificationQueue.new_notification(notify_type, notify_id)
 
+    @AuthRequired()
```

Comment:
If you add this to `ApiRoot`, how are you allowed to make a `POST` request to `/api/auth` to login in the first place?

Reply:
The tests did pass!